halyard-config/src/main/java/com/netflix/spinnaker/halyard/model/v1/providers/Provider.java

```diff
@@ -22,7 +22,7 @@
 import java.util.List;
 
 @Data
-public class Provider<T extends Account> {
+public class Provider<T extends Account> implements Cloneable {
```

Comment:
Yup, but this approach let's annotate explicitly pick what `Account` each provider is using.